charts/atlas-cluster/templates/atlas-project.yaml

```diff
@@ -0,0 +1,23 @@
+# {{- if .Values.project.enable }}
+apiVersion: atlas.mongodb.com/v1
+kind: AtlasProject
+metadata:
+  name: {{ include "atlas-cluster.fullname" . }}-{{ .Values.project.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "atlas-cluster.labels" . | nindent 4 }}
+  annotations:
+    'helm.sh/hook': post-delete,pre-install,pre-upgrade
+spec:
+  name: {{ .Values.project.atlasProjectName }}
+  connectionSecretRef:
+{{- if .Values.atlas.connectionSecretName }}
+    name: {{ .Values.atlas.connectionSecretName}}
+{{- else }}
+    name: {{ include "atlas-cluster.fullname" . }}-secret
```

Comment:
do we want to handle the scenario with global operator secrets? 
BTW I think we can support the global operator secret in the Operator chart.